modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -2442,6 +2442,15 @@ export const console: ConsoleNS = {
                                     empty: "මෙය අත්‍යවශ්‍ය ක්ෂේත්‍රයකි.",
                                     invalid: "මෙය වලංගු URL එකක් නොවේ"
                                 }
+                            },
+                            replyToLogout: {
+                                hint: "පිටවීමේදී ප්‍රතිචාරය හසුරුවන ආර්පී එන්ඩ්පොයින්ට් URL ඇතුල් කරන්න.",
```

Comment:
Shall we add the proper sinhala translation for RP endpoint here?

Reply:
Added with [0f5a444](https://github.com/wso2/identity-apps/pull/4518/commits/0f5a444d9f777190086315c82c4099bd5b6dd961)

Reply:
thanks